setup.py

```diff
@@ -20,11 +20,26 @@
 
 import semantic_version
 
-from config import appcmdname, applongname, appname, appversion, copyright, update_feed, update_interval
+from config import (
+    appcmdname, applongname, appname, appversion, copyright, git_shorthash_from_head, update_feed, update_interval
+)
```

Comment:
This is >< close to needing to be multi-line due to line length, but isort complains if *this* is multi-line.